hummingbot/connector/derivative/kucoin_perpetual/kucoin_perpetual_derivative.py

```diff
@@ -154,14 +154,7 @@ def start(self, clock: Clock, timestamp: float):
 
     def _is_request_exception_related_to_time_synchronizer(self, request_exception: Exception):
         error_description = str(request_exception)
-        ts_error_target_str = self._format_ret_code_for_print(ret_code=CONSTANTS.RET_CODE_AUTH_TIMESTAMP_ERROR)
-        param_error_target_str = (
-            "KC-API-TIMESTAMP Invalid -- Time differs from server time by more than 5 seconds"
-        )
-        is_time_synchronizer_related = (
-            ts_error_target_str in error_description or param_error_target_str in error_description
-        )
-        return is_time_synchronizer_related
+        return CONSTANTS.RET_CODE_AUTH_TIMESTAMP_ERROR in error_description and "KC-API-TIMESTAMP" in error_description

```

Comment:
nice catch! did you check if also appears in kucoin spot?
If so you can add it to this PR

Reply:
Weirdly enough the method is not implemented at all there but I'm 99% sure (looking at the documentation) that the error code and message will be the same. I will add it to this PR.
I will also fix the lint issues, I see the build is failing.

Reply:
just approved the run